mozilla/foundation.mozilla.org
# added snippet code to foundation site

# Description
Related PRs/issues: #9726 

This PR introduces the inline JS snippet that is needed in order to render the Fundraiseup modal whenever a user visits the URL http://localhost:8000/en/?form=FUNWTEYJTEM.

# Screenshots

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/18314510/203667729-5606bc5b-9f2d-4ccc-ac14-3f7234cfec85.png">

# Steps to test

1. Check this branch out
2. Update your CSP values in .env with the values found [here](https://help.fundraiseup.com/can-fundraise-up-be-used-on-a-site-with-a-csp)
3. Visit http://localhost:8000/en/?form=FUNWTEYJTEM and the Fundraiseup modal should render as expected!

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**
